webpack.config.js

```diff
@@ -105,8 +109,39 @@ module.exports = (env = 'development') => {
       ],
       ServiceWorker: {navigateFallbackURL: '/'},
     }),
+    new webpack.DefinePlugin({
+      'process.env': {
+        NODE_ENV: JSON.stringify(isProduction ? 'production' : 'development'),
+      },
+    }),
```

Comment:
I don’t think this is needed—we [already set `NODE_ENV` using the `EnvironmentPlugin`](https://github.com/ajgreenb/popcode/blob/1b29bb9ee61c613923c55112cdc48281ddecd560/webpack.config.js#L74) (note that the values given there are just defaults in case the key isn’t defined in `process.env` at compile time).